### DISCUSSION TITLE
Thumbnail size option

### DISCUSSION THREAD

**Discussion by jamesbradley** [lawand-dot-io/taskbar#81] [Feature requests]
Feature: Add the ability to change the thumbnail size that appears when you hover over a menubar item. 

On other task menu apps I usually turn this up pretty high so I can spot the specific window I'm after, such as browser windows with different sets of tabs open.

**Comment by lawand-dot-io** [comment]
Interesting idea. will keep it in mind

**Comment by lawand-dot-io** [comment]
This is now implemented in the latest update of Taskbar (1.3.0) which is now released :)
